container_restart_without_docker.md

```diff
@@ -0,0 +1,133 @@
+# `container_restart` without Docker
+
+## Problem
```

Comment:
<img width="361" alt="Screen Shot 2019-07-12 at 11 23 27 AM" src="https://user-images.githubusercontent.com/6332648/61139563-8df26200-a497-11e9-9e9d-e7be37c4b03f.png">
in case it's helpful, here's the graph of  # users running restart_container total # live_update users (though a lot of this is probably us running servantes?? should we come up with a way to remove ourselves from metrics??!)